recipes-kernel/linux/linux-yocto-onl_5.15.bb

```diff
@@ -6,13 +6,13 @@ KCONF_BSP_AUDIT_LEVEL = "1"
 
 LIC_FILES_CHKSUM = "file://COPYING;md5=6bc538ed5bd9a7fc9398086aedcd7e46"
 
-LINUX_VERSION ?= "5.15.64"
+LINUX_VERSION ?= "5.15.75"
 # https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/log/?h=linux-5.15.y
-SRCREV_machine ?= "1ded0ef2419e8f83a17d65594523ec3aeb2e3d0f"
+SRCREV_machine ?= "bd8a595958a5b02e58cdd6fed82d4ebc77b1988a"
 
 # Use commit for kver matching (or close to) LINUX_VERSION
 # https://git.yoctoproject.org/yocto-kernel-cache/log/kver?h=yocto-5.15
-SRCREV_meta ?= "8fa8d5e477034473ed7532fe20006b5f2bc87aa0"
+SRCREV_meta ?= "f52b541b6cc95d847e13ef42b133cb0fee8141a9"
```

Comment:
There is no specific bump vo kver 5.15.75 in git.yoctoproject.org/yocto-kernel-cache. There is however a commit "kver: bumping to v5.15.76" a120c990509eccdaa613b264a4f6c187277548df . Maybe we should bump to this one and the tag "v5.15.76" on commit 4f5365f77018349d64386b202b37e8b737236556 in linux respectively ?

Reply:
Given how slow the yocto git server is today, I likely won't be able to test any changes in a meaningful way.